src/authentication/oauth2Auth.ts

```diff
@@ -167,6 +167,8 @@ export class Oauth2Auth extends AlfrescoApiClient {
             if (this.config.oauth2.silentLogin && !this.isPublicUrl()) {
                 this.implicitLogin();
             }
+            let accessToken = this.storage.getItem('access_token');
```

Comment:
How does it play with the following situation:
token is not valid anymore => implicitLogin redirects => we might still set the accessToken?

Reply:
maybe I can add a validation before good point